src/metabase/lib/core.cljc

```diff
@@ -161,7 +161,8 @@
   lower]
  [lib.fe-util
   expression-clause
-  expression-parts]
+  expression-parts
+  filter-args-display-name]
```

Comment:
This functions should be used when rendering static viz stuff, right? Does it still count as a FE utility function?

Reply:
I don't believe so, in static viz the timeseries widgets are not shown.
![New question-11_21_2023, 10_15_29 AM](https://github.com/metabase/metabase/assets/105012/4ba319fe-00fb-4029-a5dd-320350b8dbcc)